src/main/java/com/aws/greengrass/logmanager/LogManagerService.java

```diff
@@ -506,11 +510,17 @@ private void processCloudWatchAttemptLogInformation(Map<String, Set<String>> com
                                                         String fileName,
                                                         CloudWatchAttemptLogFileInformation
                                                                 cloudWatchAttemptLogFileInformation) {
-        File file = new File(fileName);
+        LogFile file = new LogFile(fileName);
         String componentName = attemptLogInformation.getComponentName();
+        boolean isActiveFile = false;
+        if (ACTIVE_LOG_FILE_FEATURE_ENABLED_FLAG) {
+            LogFileGroup logFileGroup = attemptLogInformation.getLogFileGroup();
+            // TODO: the following logic is only for passing this small PR while not changing the current context
+            isActiveFile = logFileGroup.isActiveFile("");
```

Comment:
Any reason we would want this in here. It would always return `false` right?

Reply:
We have not got the fileHash in this commit yet, so leave "" instead.